src/impl_methods.rs

```diff
@@ -154,12 +154,12 @@ where
 
     /// Return an uniquely owned copy of the array.
     ///
-    /// If the input array is contiguous and its strides are positive, then the
-    /// output array will have the same memory layout. Otherwise, the layout of
-    /// the output array is unspecified. If you need a particular layout, you
-    /// can allocate a new array with the desired memory layout and
-    /// [`.assign()`](#method.assign) the data. Alternatively, you can collect
-    /// an iterator, like this for a result in standard layout:
+    /// If the input array is contiguous, then theoutput array will have the same
```

Comment:
Also, I don't want to guarantee that: "Otherwise, the layout of the output array is standard C." I'd much rather leave the sentence as-is: "Otherwise, the layout of the output array is unspecified." In the future, we may change `.to_owned()` to more intelligently choose the layout of the output array for the non-contiguous-input cases.

Reply:
Ok, got it